app/src/main/java/app/revanced/integrations/sponsorblock/SponsorBlockUtils.java

```diff
@@ -541,18 +541,16 @@ public static void importSettings(String json, Context context) {
                 editor.putString(category.key, behaviour.key);
             }
 
+            SettingsEnum.SB_UUID.saveValue(settingsJson.getString("userID"));
+            SettingsEnum.SB_IS_VIP.saveValue(settingsJson.getBoolean("isVip"));
             SettingsEnum.SB_SHOW_TOAST_WHEN_SKIP.saveValue(!settingsJson.getBoolean("dontShowNotice"));
             SettingsEnum.SB_SHOW_TIME_WITHOUT_SEGMENTS.saveValue(settingsJson.getBoolean("showTimeWithSkips"));
-            SettingsEnum.SB_COUNT_SKIPS.saveValue(settingsJson.getBoolean("trackViewCount"));
-            SettingsEnum.SB_IS_VIP.saveValue(settingsJson.getBoolean("isVip"));
             SettingsEnum.SB_MIN_DURATION.saveValue(Float.valueOf(settingsJson.getString("minDuration")));
-            SettingsEnum.SB_UUID.saveValue(settingsJson.getString("userID"));
-            SettingsEnum.SB_LAST_VIP_CHECK.saveValue(settingsJson.getLong("lastIsVipUpdate"));
-
+            SettingsEnum.SB_COUNT_SKIPS.saveValue(settingsJson.getBoolean("trackViewCount"));
 
             String serverAddress = settingsJson.getString("serverAddress");
             if (serverAddress.equalsIgnoreCase("https://sponsor.ajay.app")) {
-                serverAddress = (String) SettingsEnum.SB_API_URL.getDefaultValue();
+                serverAddress = (String) SettingsEnum.SB_API_URL.getDefaultValue(); // upgrade from old url
```

Comment:
What do you mean by that?

Reply:
the logic was already there.  The browser extension gives the url 'https://sponsor.ajay.app'
but ReVanced integration wants to use "https://sponsor.ajay.app/api/"

I'll clarify the comment.

Reply:
Shouldn't we migrate to `https://sponsor.ajay.app/` then?

Reply:
yeah probably.  But my goal here is just to fix the bugs.

Reply:
I just checked again, and the browser extension does not accept importing "http://server/api/"

I changed ReVance to behave the same, and it import/exports only the host name ("http://server")